CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.83.0 / 2025-05-27
+
+* [FEATURE] Add `limits` option for Alertmanager silences. #7478
+* [FEATURE] Add `NoTranslation` as a `translationStrategy` option to OTLP config. #7539
+* [FEATURE] Add `nameEscapingScheme` option to Prometheus CRDs. #7538
+* [FEATURE]: Add `convertHistogramsToNHCB` field to OTLPConfig #7533
+* [FEATURE]: Support add `convert_classic_histograms_to_nhcb` option for prometheus global config #7543
+* [FEATURE]: Add new `MetricNameEscapingScheme` and `MetricNameValidationScheme` parameters to ScrapeConfig #7555
+* [ENHANCEMENT] Do not insert sharding relabeling in scrapeConfigs if already present.  #7421
```

Comment:
I think this should be BUGFIX?

Reply:
Honestly, I am not sure if that is the behavior we expected. I did think it could be a bug fix, but went with what the author of the PR mentioned in the description. Would you prefer BUGFIX instead?

Reply:
ok it looks to be enhancement. I misread